src/lib/svg_text_utils.js

```diff
@@ -755,6 +755,16 @@ function alignHTMLWith(_base, container, options) {
     };
 }
 
+exports.makeTextShadow = function(offset, color) {
```

Comment:
Now that `offset` is always 1, let's simplify this function:
```js
exports.makeTextShadow = function(color) {
    return (
       '1px 1px 1px' + color + ', ' +
       '-1px -1px 1px' + color + ', ' +
       '1px -1px 1px' + color + ', ' +
       '-1px 1px 1px' + color
   );
};
```
But wait why do we need these four corners?

Reply:
See discussion in https://www.codesdope.com/blog/article/adding-outline-to-text-using-css/